prevodach.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios').default;
+const { v4: uuidv4 } = require('uuid');
+
+// PLEASE DONT STEAL THIS
+// GCP SECRET VALUE ?
+const { translatorKey } = require('config.json');
```

Comment:
```suggestion
const { translatorKey } = require('./config.json');
```

I think you need to do a relative path, otherwise it will think it's a node_module package. My bad for confusing earlier

Reply:
Just fyi the code you sent me ("2378d39fb69d420tod69fd20228d4ca8") is stored with the key "swekey", if that makes a difference

Reply:
it makes the difference in that he needs to change `translatorKey` -> `swekey` (unless of course you change it to `translatorKey`. 

Also, the reason the the key is in a hidden file is so that it won't be publically available for anyone to use, which it now is (unless you delete your message)

Reply:
```suggestion
const { swekey } = require('config.json');
```